src/ut/utils_test.cpp

```diff
@@ -198,3 +219,527 @@ TEST_F(UtilsTest, ParseIPAddresses)
   EXPECT_FALSE(parse_ip_target("1.2.3.4:8888", ai.address));
 }
 
+TEST_F(UtilsTest, Split)
```

Comment:
What's the current coverage for utils now? Is it worth moving it out of coverage-not-yet and adding LCOVs for the places that still aren't covered.

Reply:
Still not very high. I'm going to leave it in coverage-not-yet. 

```
utils.cpp                                    325     236    72%   53,59,65-66,406-407,442,444,452,454,457-459,461-462,465-467,469,573,579,581,584,586-587,590,593-594,598-601,605,608,611-613,616,618,621,623,626-627,629,631,634,638,640,642,644,646,648,652,654,658,661-662,664,666,669,672,675,684-686,688,697,703,705,709,711-713,717,720,722-723,727,729,731,734-735,757,759,818,824-825,828,831
```